src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -27,6 +26,25 @@ const Container = styled(Box)(() => ({
   overflowY: 'auto',
 }));
 
+/**
+ * Converts the "amount" string to a number to remove ".0"
+ * If the value is 0 or isExpense is true, it returns the value as is.
+ * Otherwise, it removes the '-' character if present, or prepends it if absent.
+ */
+const formatAmountForExport = (
+  amount?: string | null,
+  isExpense?: boolean,
+): number => {
+  if (!amount) {
+    return 0;
+  }
+
+  if (amount === '0' || isExpense) {
+    return Number(amount);
+  }
+  return amount?.[0] === '-' ? Number(amount.substring(1)) : -Number(amount);
```

Comment:
Is there a reason this isn't just `-Number(amount)`? Because either way we're flipping the sign of `amount`, right?

Reply:
Good catch!